Make sure the right strip is used regardless of the environment.

--- gcc-gcc-7.5.0-oi-0/install-sh.orig	2021-08-17 16:01:03.765264739 +0000
+++ gcc-gcc-7.5.0-oi-0/install-sh	2021-08-17 15:59:20.045587895 +0000
@@ -62,7 +62,7 @@
 mkdirprog=${MKDIRPROG-mkdir}
 mvprog=${MVPROG-mv}
 rmprog=${RMPROG-rm}
-stripprog=${STRIPPROG-strip}
+stripprog="/usr/bin/strip -x"
 
 posix_mkdir=
 
